src/overlays/actors/ovl_En_An/z_en_an.c

```diff
@@ -13,160 +20,3441 @@
+
+        case 0x2:
+            EnAn_ChangeAnim(this, play, ENAN_ANIM_TASTE_END);
+            this->msgEventState++;
+            break;
+
+        default:
+            break;
+    }
+
+    return ret;
+}
+
+s32 EnAn_MsgEvent_LaundryPool(Actor* thisx, PlayState* play) {
+    EnAn* this = THIS;
+    s32 ret = false;
+
+    switch (this->msgEventState) {
+        case 0x0:
+            if ((Player_GetMask(play) == PLAYER_MASK_KAFEIS_MASK) ||
+                CHECK_WEEKEVENTREG(WEEKEVENTREG_TALKED_ANJU_IN_LAUNDRY_POOL)) {
+                this->msgEventState++;
+                goto label;
+            } else {
+                ret = true;
+                this->stateFlags |= ENAN_STATE_DRAW_KAFEI_MASK;
+                this->msgEventState++;
+            }
+            break;
+
+        case 0x1:
+        label:
+            this->stateFlags &= ~(ENAN_STATE_ENGAGED | ENAN_STATE_DRAW_KAFEI_MASK);
+            this->stateFlags |= ENAN_STATE_LOST_ATTENTION;
+            EnAn_ChangeAnim(this, play, ENAN_ANIM_UMBRELLA_CRY);
+            ret = true;
+            this->msgEventState++;
+            break;
+
+        default:
+            break;
+    }
+
+    return ret;
+}
+
+s32* EnAn_GetMsgEventScript(EnAn* this, PlayState* play) {
+    Player* player = GET_PLAYER(play);
+
+    switch (this->scheduleResult) {
+        case ANJU_SCH_GIVE_LUNCH_TO_GRANNY:
+            this->msgEventFunc = EnAn_MsgEvent_GiveLunchToGranny;
+            return sAnjuMsgScript_SchGiveLunchToGranny;
+
+        case ANJU_SCH_RECEIVE_LETTER_FROM_POSTMAN:
+            this->msgEventFunc = EnAn_MsgEvent_ReceiveLetterFromPostman;
+            return sAnjuMsgScript_SchReceiveLetterFromPostman;
+
+        case ANJU_SCH_ATTEND_GORON:
+            this->msgEventFunc = EnAn_MsgEvent_AttendGoron;
+            return sAnjuMsgScript_SchAttendGoron;
+
+        case ANJU_SCH_COOKING:
+            this->msgEventFunc = EnAn_MsgEvent_Cooking;
+            return sAnjuMsgScript_SchCooking;
+
+        case ANJU_SCH_RANCH:
+            return sAnjuMsgScript_SchRanch;
+
+        case ANJU_SCH_WITH_KAFEI:
+            return sAnjuMsgScript_SchWithKafei;
+
+        default:
+            break;
+    }
+
+    if (player->transformation == PLAYER_FORM_DEKU) {
+        switch (this->scheduleResult) {
+            case ANJU_SCH_WAITING_FOR_KAFEI:
+                return sAnjuMsgScript_SchWaitingForKafei;
+
+            case ANJU_SCH_WALKING_40:
+            case ANJU_SCH_WALKING_42:
+            case ANJU_SCH_WALKING_43:
+            case ANJU_SCH_WALKING_44:
+            case ANJU_SCH_WALKING_45:
+            case ANJU_SCH_WALKING_47:
+                return sAnjuMsgScript_80B58B7C;
+
+            case ANJU_SCH_LAUNDRY_POOL_SIT:
+                return sAnjuMsgScript_SchLaundryPoolDeku;
+
+            case ANJU_SCH_WALKING_49:
+                return sAnjuMsgScript_80B589FC;
+
+            default:
+                break;
+        }
+
+        return sAnjuMsgScript_DekuDefault;
+    }
```

Comment:
```suggestion
            default:
                return sAnjuMsgScript_DekuDefault;
        }
    }
```
Could do something like this: